flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/batch/sql/AggregateITCase.scala

```diff
@@ -559,4 +559,57 @@ class AggregateITCase(
 
     TestBaseUtils.compareResultAsText(result.asJava, expected)
   }
+
+  @Test
+  def testMultipleDistinctWithDiffParams(): Unit = {
+    val env = ExecutionEnvironment.getExecutionEnvironment
+    val tEnv = TableEnvironment.getTableEnvironment(env, config)
+    val sqlQuery =
+      "SELECT b, " +
+      "  SUM(DISTINCT (a / 3)), " +
+      "  COUNT(DISTINCT SUBSTRING(c FROM 1 FOR 2))," +
+      "  COUNT(DISTINCT c) " +
+      "FROM MyTable " +
+      "GROUP BY b " +
+      "ORDER BY b"
+
+    val data = new scala.collection.mutable.MutableList[(Int, Long, String)]
```

Comment:
Replace this with `CollectionDataSets.get5TupleDataSet(env)`.
I think we can add a `case when` to generate null data. For example:
```
    val t = CollectionDataSets.get5TupleDataSet(env).toTable(tEnv).as('a, 'b, 'c, 'd, 'e)
    tEnv.registerTable("MyTable", t)
    val sqlAddNull = "SELECT a, b, c, e, CASE d WHEN 'Hallo' THEN null ELSE d END AS d From MyTable"
    val sqlQuery =
      "SELECT d, " +
      "  COUNT(DISTINCT d), " +
      "  COUNT(DISTINCT e) " +
      s"FROM ($sqlAddNull) " +
      "GROUP BY d " +
      "ORDER BY d"
```

Reply:
updated, thank you very much for review and suggestions~, @hequn8128